src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,53 @@
+.catalog {
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 200px;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    margin-bottom: 48px;
+  }
```

Comment:
I hope that the card can be a BEM block.

It would be beneficial to move all of the styles, except for the margins that are necessary for positioning the card, to the .card BEM block.

Reply:
I have 'catalog_card' as the element of 'catalog' BEM block, so I think 'card' cant be separate BEM block, but I can make mixin for 'card' positioning